pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -151,6 +158,19 @@ resources:
     expression: "0 0 * * *"
     fire_immediately: true
 
+- name: census-rm-kubernetes-repo-ddl
+  type: git
+  source:
+    uri: git@github.com:ONSdigital/census-rm-kubernetes.git
+    private_key: ((github.service_account_private_key))
+    paths: [optional/census-rm-ddl-pod.yml]
+
+- name: midnight
```

Comment:
we already have an `every-midnight` trigger job

Reply:
Sorry forgot you added one with your card 👍